eos/form-factors/mesonic-processes.hh

```diff
@@ -79,6 +79,15 @@ namespace eos
         static constexpr const double mR2_0p = 5.540 * 5.540; // B_{u,d} scalar: M(B_s scalar) - M(B_s^*) + M(B_{u,d}^*)
         static constexpr const double tp = (m_B + m_P) * (m_B + m_P);
         static constexpr const double tm = (m_B - m_P) * (m_B - m_P);
+        // zero of the conformal mapping: z(t0, t0) = 0.0
+        // This optimal value follows from z(0, t0) = - z(tm, t0)
+        static constexpr double t0 = 14.7;
```

Comment:
Why are these values hard-coded? They are supposed to be parameters in the BFW parameterization

Reply:
I agree, I overlooked it.

Reply:
These value are added in addition to the BFW parametrization. Does not hurt at all, the parameters still exist and are used!

Reply:
But t0 is not and should not be used here, right?